lib/reference/mediaTypes.mjs

```diff
@@ -0,0 +1,14 @@
+/**
+ * @see https://drafts.csswg.org/mediaqueries/#ref-for-media-type①⑨
+ * @type {ReadonlySet<string>}
+ */
+export const deprecatedMediaTypesNames = new Set([
+	'tty',
+	'tv',
+	'projection',
+	'handheld',
+	'braille',
+	'embossed',
+	'aural',
+	'speech',
```

Comment:
Does it make sense to sort the words alphabetically?

Reply:
I just copied this list from the [specification](https://drafts.csswg.org/mediaqueries-5/#media-types) (we added a link to it in the docs).

Maybe it would be worth sorting it. On the other hand, it might actually be easier to search for a similar list directly in the specification text. In general, let's do as you think is best.

Reply:
Sorted